Harshit_Maharshi.java

```diff
@@ -0,0 +1,26 @@
+import java.io.*;
+import java.util.*;
+class Harshit_Maharshi {
+	public static void main(String args[]) throws Exception {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		String str = br.readLine();
+		int ch, i, n = Integer.parseInt(br.readLine());
+		String[] m = new String[n];
+		int[] j = new int[n];
+		Arrays.fill(j,-1);
+		for(i=0;i<n;i++)
```

Comment:
Please use a separate method to get the output & give proper names to the local variables. And please do proper indentation(format) in your code.